src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.cpp

```diff
@@ -0,0 +1,438 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/WrappedGr.hpp"
+
+#include "DataStructures/DataBox/Prefixes.hpp"  // IWYU pragma: keep
+#include "DataStructures/DataVector.hpp"        // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"     // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrSchild.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp"
+#include "PointwiseFunctions/GeneralRelativity/ComputeSpacetimeQuantities.hpp"
+#include "Utilities/GenerateInstantiations.hpp"
+// IWYU pragma: no_forward_declare ::Tags::deriv
+
+/// \cond
+namespace GeneralizedHarmonic {
+namespace Solutions {
+template <typename SolutionType>
```

Comment:
@geoffrey4444 here you need:
```cpp
#define FUNC_DEF(r, data, elem)                                            \
  template <typename SolutionType>                                         \
  auto WrappedGr<SolutionType>::variables(                                 \
      const tnsr::I<DataVector, GeneralizedHarmonic::Solutions::WrappedGr< \
                                    SolutionType>::volume_dim>& /*x*/,     \
      double /*t*/, tmpl::list<elem> /*meta*/,                             \
      const IntermediateVars& intermediate_vars)                           \
      const noexcept->tuples::TaggedTuple<elem> {                          \
    return {get<elem>(intermediate_vars)};                                 \
  }

#define MY_LIST                                                              \
  BOOST_PP_TUPLE_TO_LIST(                                                    \
      11,                                                                    \
      (gr::Tags::Lapse<DataVector>, TimeDerivLapse, DerivLapse, TagShift,    \
       TimeDerivShift, DerivShift, TagSpatialMetric, TimeDerivSpatialMetric, \
       DerivSpatialMetric, TagInverseSpatialMetric, TagExCurvature,          \
       gr::Tags::SqrtDetSpatialMetric<DataVector>))

  BOOST_PP_LIST_FOR_EACH(FUNC_DEF, _, MY_LIST)
#undef MY_LIST
#undef FUNC_DEF
```
I was only able to get this working with the preprocessor, not anything else :man_shrugging: